build-ci-wheels/action.yml

```diff
@@ -0,0 +1,77 @@
+name: "Build a c-extension Python library"
+description: "Build a c-extension Python library by generating manylinux wheels"
+
+inputs:
+  CPython:
+    description: 'Desired CPython version'
+    default: '3.10'
+    required: false
+    type: string
+  cibw_build:
+    description: 'Desired build wheel cpython version'
+    default: "cp310-*"
+    required: false
+    type: string
+  cibw_skip:
+    description: 'Desired build wheel for skipping'
+    default: "*-musllinux* "
+    required: false
+    type: string
+  cibw_archs:
+    description: 'Desired build arhitecture'
+    default: " auto "
+    required: false
+    type: string
+  requires-pypy:
+      description: 'Whether requires build wheel for PyPy'
+      default: false
+      required: false
+      type: boolean
```

Comment:
The notation we are following or input variables is:

* Use dash for input variables. Do `cibw-build` instead of `cibw_build`.
* Use lowercase for input variables. Do `cpython-version` instead of `Cpython`.